IntroducingMethods/src/cst1201/IntroducingMethods.java

```diff
@@ -3,42 +3,103 @@
 import java.util.Scanner;
 
 /**
- * TODO: Description of your class.
+ * CST 1201
  *
- * @author TODO: Your name here.
+ * @author JianLang Lin,Ziying Guo
  */
 public class IntroducingMethods {
 
     public static void main(String[] args) {
+        Scanner keyboard = new Scanner(System.in);
+        System.out.print("Enter a string: ");
 
-        // TODO: Fill in the body with your code
+        getInputString(keyboard);
```

Comment:
There should be more stuff here. See https://github.com/cst1201-d644/Introducing-Methods/blob/solution/IntroducingMethods/test/cst1201/IntroducingMethodsTest.java.